### ISSUE TITLE
Insufficient contrast ratio 

### ISSUE THREAD

**Issue by Tejaswini411** [smeggins/aloha#2]
Tool to test color values.
website: https://webaim.org/resources/contrastchecker/ might help to get the correct contrast ratio.
